src/DynamoCoreWpf/Views/Preview/Watch3DView.xaml.cs

```diff
@@ -302,6 +290,9 @@ private void SetupScene()
             SetCameraToDefaultOrientation();
 
             DrawGrid();
+
+            renderTechnique = new RenderTechnique("RenderDynamo");
```

Comment:
Here we create our custom `RenderTechnique` and we register the event handler to register this effect when Helix is initializing.